pkg/virt-operator/util/config.go

```diff
@@ -419,6 +426,22 @@ func (c *KubeVirtDeploymentConfig) GetVerbosity() string {
 	return "2"
 }
 
+func (c *KubeVirtDeploymentConfig) GetControllerFlags() map[string]string {
+	return c.ControllerFlags
+}
+
+func (c *KubeVirtDeploymentConfig) GetControllerFlagsAsArray() []string {
+	flags := make([]string, 0)
+
+	for f, v := range c.GetControllerFlags() {
+		str := matchAllCap.ReplaceAllString(f, "${1}-${2}")
+		flags = append(flags, fmt.Sprintf("--%s", strings.ToLower(str)))
+		flags = append(flags, v)
```

Comment:
what happens if a boolean flag is added? Then there wouldn't be a value to set it to true, right?

maybe ignore appending the value to the slice `if v == ""` ?